docs/csharp/language-reference/keywords/accessibility-levels.md

```diff
@@ -21,9 +21,9 @@ Use the access modifiers, [public](../../../csharp/language-reference/keywords/p
 |`public`|Access is not restricted.|  
 |`protected`|Access is limited to the containing class or types derived from the containing class.|  
 |`internal`|Access is limited to the current assembly.|  
-|`protected internal`|Access is limited to the current assembly or types derived from the containing class.|  
+|[`protected internal`](protected-internal.md)|Access is limited to the current assembly or types derived from the containing class.|  
 |`private`|Access is limited to the containing type.|  
-|`private protected`|Access is limited to the containing class or types derived from the containing class within the current assembly. Available since C# 7.2. |  
+|[`private protected`](private-protected.md)|Access is limited to the containing class or types derived from the containing class within the current assembly. Available since C# 7.2. |  
```

Comment:
@svick instead of adding the link here, should we perhaps add the links for those on line 28 to entries look cohesive on the table?

![image](https://user-images.githubusercontent.com/12971179/35605315-e52c3036-05fc-11e8-8319-9613b6b430ee.png)

Reply:
You're right that it doesn't look good like this.

But considering that the linked articles are actually about the accessibility levels and not about the access modifiers, I have moved all the links into the table instead. And I also made equivalent change to access-modifiers.md. What do you think?